OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -237,6 +240,15 @@ $ sudo chown 1000:1000 /scratch/wcpdocker/volumes/es
 
 #### B. Creating and Running Elasticsearch Container
 
+>Note: With the beginning of April 2022 PSU release of Oracle WebCenter Portal, to install Elasticsearch you need to download the Elasticsearch 7.17.x binaries from Elasticsearch [Releases](https://www.elastic.co/downloads/past-releases#elasticsearch).
+
+Update the file name as `ELASTIC_SEARCH_INSTALLER_FILE_NAME` value and Elasticsearch version as `ELASTIC_SEARCH_VERSION` value.
+
+Copy the downloaded Elasticsearch binary to host mount directory (`DATA_MOUNT`) using below command. 
+```
+$  cp elasticsearch-7.17.x-linux-x86_64.tar.gz /scratch/wcpdocker/volumes/wcpportal
+```
+
 ```
 $ docker run -i -t --name <ES_CONTAINER_NAME> --network=<NETWORK_NAME> -p <HostFreePort>:9200 --volumes-from <WCP_CONTAINER_NAME> -v <ES_DATA_MOUNT>:/u01/esHome/esNode/data --env-file webcenter.env.list oracle/wcportal:12.2.1.4 configureOrStartElasticsearch.sh
 ```
```

Comment:
What is this command for? There is no context here.

Reply:
Added the context